### PR TITLE
[ Feat ] dropdown for gitlab stage status & platforms reduced to 3 .

### PR DESCRIPTION
**Pull request by bhaskarhc**
**What this PR does / why we need it**:
  -  Displaying :tv:  **Gitlab stages** is now more detailed :+1: , For each gitlab-stage  success  , failure & skip count displayed by clicking dropdown icon :arrow_down:  .
  - Non-build pipelines are removed :no_entry_sign: , Now GKE , AKS & EKS platforms are present .

screenshot from 2019-01-16 13-07-17

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
     <link rel="icon" type="image/x-icon" href="favicon.ico">
     <!-- Favicon and touch icons -->
     <link rel="icon" type="image/x-icon" href="https://raw.githubusercontent.com/openebs/openebs/master/documentation/source/_static/favicon.ico">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js" integrity="sha384-b/U6ypiBEHpOf/4+1nzFpr53nxSS+GLCkfwBdFNTxtclqqenISfwAzpKaMNFNmj4" crossorigin="anonymous"></script>
```
Comment (inyee786) [inline src/index.html]: why you are using cdn, even we have npm package

File: src/app/table/table.component.html [hunk 2]
```diff
@@ -56,15 +56,15 @@ <h3 *ngIf="undefined !== restItems" style="color:white">Builds in last one week:
                   <span style="pointer-events: none;" class="mx-4"><img style="padding:2px;" height="26px;" src="/assets/images/cloud/eks.svg">
                     <p style="font-size: 13px;">EKS</p>
                   </span>
-                  <span style="pointer-events: none;" class="mx-4"><img style="padding:2px;" height="26px;" src="/assets/images/cloud/packet.svg">
+                  <!-- <span style="pointer-events: none;" class="mx-4"><img style="padding:2px;" height="26px;" src="/assets/images/cloud/packet.svg">
```
[the remark this answers is not in the document]
Comment (inyee786) [reply src/app/table/table.component.html]: we can delete it know, if required will get it back, remove all unused code

[the remark this answers is not in the document]
Comment (bhaskarhc) [reply src/app/table/table.component.html]: @inyee786  i deleted unused code !

**Comment by inyee786** [comment]
@bhaskarhc  add smooth css transition for dropdown

**Pull request merged**